app/views/pages/home.html.erb

```diff
@@ -1,2 +1,3 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
```

Comment:
same here

Reply:
these have been already deleted. Hence the - sign and red color